.github/workflows/scripts/create-lambda-function.sh

```diff
@@ -7,33 +8,43 @@ REGISTRY="$4"
 IMAGE="$5"
 ROLE_ARN="$6"
 
-# Check if the function exists already
-# If it does, update it with the new code from the PR
-if aws lambda get-function --function-name "$FUNCTION_NAME-$PR_NUMBER-$LANG" > /dev/null 2>&1; then
-  aws lambda update-function-code \
-    --function-name "$FUNCTION_NAME-$PR_NUMBER-$LANG" \
-    --image-uri "$REGISTRY"/"$IMAGE":"$PR_NUMBER" > /dev/null 2>&1
+FULL_FUNCTION_NAME="${FUNCTION_NAME}-${PR_NUMBER}-${LANG}"
+IMAGE_TAG="${PR_NUMBER}"
+
+echo "Deploying function: $FULL_FUNCTION_NAME"
 
-# Function does not exist, create a new function and setup its function URL
+# Check if the function already exists
+if aws lambda get-function --function-name "$FULL_FUNCTION_NAME" > /dev/null 2>&1; then
+  echo "Function exists. Updating code..."
+  aws lambda update-function-code \
+    --function-name "$FULL_FUNCTION_NAME" \
+    --image-uri "${REGISTRY}/${IMAGE}:${IMAGE_TAG}"
```

Comment:
Could you add this back in - it prevents leaking the lambda function's env vars:
```suggestion
    --image-uri "${REGISTRY}/${IMAGE}:${IMAGE_TAG}" > /dev/null 2>&1
```